### PR TITLE
[ADD][stock_account_valuation_report]

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
When you trigger a report of inventory valuation, and you use
perpetual inventory, you should be able to reconcile the valuation
from an inventory perspective with the valuation
from an accounting perspective.

This module changes the report in *Inventory / Reporting / Inventory Valuation*
to display separately the Quantity and Value of each product for the
Inventory and the Accounting systems .

### CODE REVIEW

**Comment by rousseldenis** [comment]
@jbeficent @hveficent Is this still WIP?

**Comment by HviorForgeFlow** [comment]
@rousseldenis I don't think so, we are currectly using it, a review would be great. :smile:
Might tests are needed

File: stock_account_valuation_report/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# Copyright 2018 Aleph Objects, Inc.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Stock Account Valuation Report",
+    "version": "11.0.1.0.0",
+    "summary": "Improves logic of the Inventory Valuation Report",
+    "author": "Eficent, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-reporting",
+    "category": "Generic",
```
Comment (rousseldenis) [inline stock_account_valuation_report/__manifest__.py]: Maybe 'Warehouse Management' instead to help finding it easily (in store).

**Review by rousseldenis — APPROVED** [review]
Minor comment. LGTM.

Code review + functionnal.

**Comment by rousseldenis** [comment]
@jbeficent Could you squash your commits ?

**Comment by rousseldenis** [comment]
@jbeficent Maybe some tests on different computed fields ?

**Review by gurneyalex — APPROVED** [review]

**Comment by rousseldenis** [comment]
@jbeficent Is this ok ?

**Comment by LoisRForgeFlow** [comment]
We'll try to add tests in during the migration. Let's merge this now.

**Pull request merged**